pybv/io.py

```diff
@@ -31,14 +31,14 @@
 
 
 def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
-                      events=None, resolution=1e-7, unit='µV', scale_data=True,
+                      events=None, resolution=0.1, unit='µV',
```

Comment:
So this means we're ensuring a correct roundtrip for values >= 0.1 µV by default, yes? Could this be made a little bit more explicit when describing the `resolution` param?

Reply:
I actually don't get the resolution parameter beyond the fact that before saving we do "data *= (1/resolution)" and at reading, readers do "data *= resolution". I don't get what that buys us
... Can you explain that to me?

Reply:
Ohhh because we are writing to int! So when writing to float, we can completely ignore resolution!